basics.mdx

```diff
@@ -281,7 +281,7 @@ Let's test out RAG by building a **PDF Assistant** that helps us with food recip
     <CodeGroup>
 
     ```bash Mac
-    pip install -U pgvector pypdf psycopg sqlalchemy
+    pip install -U pgvector pypdf psycopg sqlalchemy psycopg-binary
```

Comment:
@stkolanu thank you, i appreciate your contribution so much. Please replace `psycopg` with `psycopg-binary`.

Also please update the windows instructions on line 288.

Thank you again.